src/index.js

```diff
@@ -0,0 +1,150 @@
+const tempUp = document.getElementById('tempUp');
+const tempDown = document.getElementById('tempDown');
+const landScape = document.getElementById("landScape");
+const cityInput = document.getElementById('cityInput');
+const cityValue = document.getElementById('cityInput').value;
+const currentCity = document.getElementById('currentCity');
+const realTemp = document.getElementById('getTemp');
+const skySelect = document.getElementById('skies');
+const sky = document.getElementById('sky').value;
+const resetCity = document.getElementById('resetCity')
+
+const updateWeather = () => {
+    tempInfo.textContent = state.temp
+};
+
+const increaseWeather = () => {
+    state.temp += 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const decreaseWeather = () => {
+    state.temp -= 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const updatecityInput = () => {
+    const updatedCity = document.getElementById('cityInput').value;
+    const currentCity = document.getElementById('currentCity');
+    state.city = updatedCity;
+    currentCity.textContent = state.city;
+};
+
+const updateresetCity = () => {
+    const updatedCity = document.getElementById('cityInput');
+    const currentCity = document.getElementById('currentCity');
+    state.city = "Seattle";
+    currentCity.textContent = state.city;
+    updatedCity.value = state.city;
+};
+
+const updateSky = () => {
+    const sky = document.getElementById('sky');
+    const skyValue = document.getElementById('skies').value;
+    if (skyValue === 'sunny') {
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+    } else if (skyValue === 'cloudy') {
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+    } else if (skyValue === 'rainy') {
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+    } else if (skyValue === 'snowy') {
+        sky.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+
+const updatelandScape = () => {
+    const sky = document.getElementById('sky');
+    if (state.temp > 80) {
+        tempInfo.style.color = 'red'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
+    } else if (state.temp > 70) {
+        tempInfo.style.color = 'orange'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'
+    } else if (state.temp > 60) {
+        tempInfo.style.color = 'yellow'
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+        landScape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'
+    } else if (state.temp > 50) {
+        tempInfo.style.color = 'green'
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+        landScape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+    } else if(state.temp < 49) {
+        tempInfo.style.color = 'teal'
+        landScape.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+const findLatitudeAndLongitude = () => {
+    let latitude, longitude;
+    axios
+    .get('https://weather-report-proxy-server.herokuapp.com/location', {
```

Comment:
Were you able to get your calls to the weather proxy to work with this URL or did you run into CORS issues? If you cloned down the proxy-server and ran it locally, then you could use `http://127.0.0.1:5000/weather` here to get the weather.

You can use a constant variable like WEATHER_URL instead of string literal here.

"Constants free the programmer from having to remember what each literal should be. Often values that stay constant throughout the program have a business meaning. If there are several such values, the programmer can define them all in the beginning of the program and then work with the easier-to-remember constant names." From: https://www.diffen.com/difference/Constant_vs_Literal